### PR TITLE
[rdu-2022] chris-evich talk title spelling correction

### PR DESCRIPTION
**Pull request by normalfaults**
[rdu-2022] chris-evich talk title spelling correction

### CODE REVIEW

File: content/events/2022-raleigh/program/chris-evich.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Talk_date = ""
 Talk_start_time = ""
 Talk_end_time = ""
-Title = "Don't Panic, It's Only a Hoard of Hungry Developers"
+Title = "Don't Panic, It's Only a Hoarde of Hungry Developers"
```
Comment (cevich) [inline content/events/2022-raleigh/program/chris-evich.md:5]: No `a` please, just `Horde`.  Thanks.

Comment (glasnt) [inline content/events/2022-raleigh/program/chris-evich.md:5]:
```suggestion
Title = "Don't Panic, It's Only a Horde of Hungry Developers"
```

**Comment by glasnt** [comment]
@cevich once the preview builds, can you confirm this change is now reflective of your talk title? Thanks!

**Comment by cevich** [comment]
I'm assuming you mean `netify/devopsdays-web/deploy-preview`?  Following that through [to the schedule page](https://deploy-preview-10863--devopsdays-web.netlify.app/events/2022-raleigh/program), it looks correct now, thanks :smiley:

**Pull request merged**